plugins/source/aws/docs/tables/aws_docdb_cluster_snapshot_attributes.md

```diff
@@ -0,0 +1,16 @@
+# Table: aws_docdb_cluster_snapshot_attributes
+
+https://docs.aws.amazon.com/documentdb/latest/developerguide/API_DBClusterSnapshotAttributesResult.html
+
+The primary key for this table is **_cq_id**.
+
+
+## Columns
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id (PK)|UUID|
+|_cq_parent_id|UUID|
+|db_cluster_snapshot_attributes|JSON|
+|db_cluster_snapshot_identifier|String|
```

Comment:
I think we need to add a `db_cluster_snapshot_arn` field here to serve as a foreign key to the parent table.